rewrite-java/src/main/java/org/openrewrite/java/AddOrUpdateAnnotationAttribute.java

```diff
@@ -46,37 +42,37 @@ public String getDisplayName() {
     @Override
     public String getDescription() {
         return "Some annotations accept arguments. This recipe sets an existing argument to the specified value, " +
-               "or adds the argument if it is not already set.";
+          "or adds the argument if it is not already set.";
```

Comment:
I think you have a custom style configured in your IDE. We use the IntelliJ internal formatter with standard configuration. 
Please reformat to keep the change as small as needed.